src/System.ComponentModel.TypeConverter/src/System.ComponentModel.TypeConverter.csproj

```diff
@@ -6,23 +6,36 @@
     <RootNamespace>System.ComponentModel.TypeConverter</RootNamespace>
     <AssemblyName>System.ComponentModel.TypeConverter</AssemblyName>
     <AssemblyVersion Condition="'$(TargetGroup)' == 'netstandard1.0' OR '$(TargetGroup)'=='net45'">4.0.0.0</AssemblyVersion>
+    <AssemblyVersion Condition="'$(TargetGroup)' == 'netstandard1.5' OR '$(TargetGroup)'=='net462'">4.1.1.0</AssemblyVersion>
```

Comment:
Once we eliminate the redundant builds most of this project file can be cleaned up.